Gemfile.lock

```diff
@@ -684,10 +685,11 @@ GEM
       childprocess (>= 0.5, < 4.0)
       rubyzip (>= 1.2.2)
     seven_zip_ruby (1.2.5)
-    simplecov (0.18.0)
+    simplecov (0.17.1)
```

Comment:
Jeje, I thought this would happen when I saw the first commit 😄

Reply:
Yeah, I had a little problem with dependencies and rebasing from master 😞

Reply:
Oh, I had problems with CodeClimate coverage report when I updated to simplecov 0.18, I thought that was the problem: https://github.com/codeclimate/test-reporter/issues/418

Reply:
Yes, we had to lock `simplecov` to v0.17.1 so that CI jobs could end successfully, see #5673 😄